lib/validation.ts

```diff
@@ -34,7 +33,7 @@ async function loadAndValidateOptions(options, shouldValidateAccount = true) {
     validAccount = await validateAccount(options);
   }
 
-  if (!(validateConfig() && validAccount)) {
+  if (!validAccount) {
```

Comment:
Was this previously running account validation before config validation? Seems like this PR will also switch the flow to validate config before account, which feels like a better order IMO